xml/schema/CRM/Aip/AipErrorLog.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+<table>
+  <base>CRM/Aip</base>
+  <class>AipErrorLog</class>
+  <name>civicrm_aip_error_log</name>
+  <comment>AIP Process</comment>
+  <log>false</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <uniqueName>aip_error_log_id</uniqueName>
+    <required>true</required>
+    <comment>Process ID</comment>
+  </field>
+  <primaryKey>
+    <name>id</name>
+    <autoincrement>true</autoincrement>
+  </primaryKey>
+
+  <field>
+    <name>process_id</name>
+    <uniqueName>aip_error_log_process_id</uniqueName>
+    <type>int unsigned</type>
+    <comment>FK to AIP Process</comment>
+    <foreignKey>
+      <name>process_id</name>
+      <table>civicrm_aip_process</table>
+      <key>id</key>
+      <onDelete>CASCADE</onDelete>
+    </foreignKey>
+    <html>
+      <type>EntityRef</type>
+    </html>
+    <pseudoconstant>
+      <table>civicrm_aip_process</table>
+      <keyColumn>id</keyColumn>
+      <labelColumn>name</labelColumn>
+      <prefetch>false</prefetch>
+    </pseudoconstant>
+  </field>
+
+  <field>
+    <name>error_timestamp</name>
+    <type>datetime</type>
+    <uniqueName>aip_error_log_error_timestamp</uniqueName>
+    <title>Last execution of this process</title>
+    <import>true</import>
+    <comment>Timestamp when the error was recorded</comment>
+    <html>
+      <type>Select Date</type>
+      <formatType>activityDateTime</formatType>
+    </html>
+  </field>
+
+  <field>
+    <name>error_message</name>
+    <type>text</type>
+    <uniqueName>aip_error_log_error_message</uniqueName>
+    <title>Error Message</title>
+    <localizable>false</localizable>
+    <comment>The error message that was recorded with the processing failure</comment>
+  </field>
+
+  <field>
+    <name>data</name>
+    <type>text</type>
+    <uniqueName>aip_error_log_error_data</uniqueName>
+    <title>Data/Record</title>
+    <serialize>JSON</serialize>
+    <localizable>false</localizable>
+    <comment>Data set (record) to be processed</comment>
+  </field>
+
+  <field>
+    <name>is_resolved</name>
```

Comment:
I think `is_scheduled` what be a more appropriate name according to the comment. Actually I don't know what either of these terms could mean with regard to a log entry.

Reply:
> I think is_scheduled what be a more appropriate name according to the comment

The intention is, that a logged error/failure could later be retried. And if that's successful, or if the issue is resolved by other means, the flag would be set to ``true``.